falco/values.yaml

```diff
@@ -201,6 +201,8 @@ driver:
       # -- Extra environment variables that will be pass onto Falco driver loader init container.
       env: {}
       args: []
+      resources: {}
+      securityContext: {}
```

Comment:
We need to explain that the `securityContext` defined  here will take precedence over the default one.

After you have done it, please run `make docs`!